### PR TITLE
update cell imageView

### PR DESCRIPTION
**Pull request by mengerpiao**
CHS:添加前边的icon,更改section间隔。

### CODE REVIEW

**Comment by mengerpiao** [comment]
CHS:添加前边的icon

File: components/Cell.js [hunk 1]
```diff
@@ -60,29 +61,42 @@ class Cell extends Component {
       rightDetailColor,
       title,
       titleTextColor,
+      iconImg,
+
     } = this.props;
 
     let { cellStyle } = this.props;
-    /** Deprecation */
+
```
Comment (Purii) [inline components/Cell.js]: Don't delete the comments.

File: components/Cell.js [hunk 2]
```diff
@@ -60,29 +61,42 @@ class Cell extends Component {
...
     // eslint-disable-next-line
     if (this.props.cellstyle) cellStyle = this.props.cellstyle;
 
     const isPressable = !!this.props.onPress;
 
-    /* Declare and merge styles with props */
```
Comment (Purii) [inline components/Cell.js]: Don't delete the comments.

File: components/Cell.js [hunk 3]
```diff
@@ -60,29 +61,42 @@ class Cell extends Component {
       rightDetailColor,
       title,
       titleTextColor,
+      iconImg,
+
     } = this.props;
 
     let { cellStyle } = this.props;
-    /** Deprecation */
```
Comment (Purii) [inline components/Cell.js]: Don't delete the comments.

File: components/Cell.js [hunk 4]
```diff
@@ -281,6 +332,29 @@ class Cell extends Component {
 }
 
 const styles = StyleSheet.create({
+
+
+
+  // cell左图片
```
Comment (Purii) [inline components/Cell.js]: Could you please write your comments in english, so that everyone could understand them?

**Pull request closed without merging**